Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -885,6 +885,16 @@ void ProgramShaderCache::CreateHeader()
     }
   }
 
+  std::string framebuffer_fetch_string = "";
+  switch (g_ogl_config.SupportedFramebufferFetch)
+  {
+  case ES_FB_FETCH_TYPE::FB_FETCH_EXT:
+    framebuffer_fetch_string = "#extension GL_EXT_shader_framebuffer_fetch: enable";
+    break;
+  case ES_FB_FETCH_TYPE::FB_FETCH_NONE:
+    framebuffer_fetch_string = "";
```

Comment:
All cases should have a `break`, regardless of whether or not they're last in a switch block.